### ISSUE TITLE
New probability for Xs = Xt for AEI training

### ISSUE THREAD

**Issue by ocastan** [taotaonice/FaceShifter#30]
The authors of the original paper have issued a new version in late June, and I noticed they corrected the training strategy of AEI and now Xs Xt are supposed to be different (and not identical anymore) 80% of the time
So same,_prob=0.8 should be changed to 0.2 on line 56 of train_AEI.py
I'm going to try this new value and I'll let you know

**Comment by tangxuvis** [comment]
hi， what is going on about your new experiments? any progresses？@ @ocastan

**Comment by tangxuvis** [comment]
i have implemented some perfomance, can we chat on email about more details? @ocastan  
my email address: tangxuvis@163.com

**Comment by ocastan** [comment]
Hello
My experiment is still running, each batch takes 3s. It has been running for about 80k iterations and the results are not very convincing so far with these new parameters.
Moreover I"ve made numerous modifications:
- gradient accumulation to simulate a batch of 24 samples with a single card
- replaced batchnorm with groupnorm because of the previous
- added gradient penalty (but it looks useless, it fires very rarely)
- replaced mean of the tensor hinge with hinge of the tensor mean (not sure it was a good idea...)
- ...
Can you tell a little bit more about your performance additions?
